pages/projects/[id].tsx

```diff
@@ -369,7 +371,14 @@ const ProjectPage: NextPage<ProjectPageProps> = props => {
 		setSuccessMsg("Success! You've uploaded a new stem to this project and become a contributor.")
 		handleUploadStemClose()
 	}
-
+	const onNewFile = (newFile: Blob) => {
+		console.log('New File Found')
+		if (!files) {
+			setFiles(() => [newFile])
+		} else {
+			setFiles(files => [...files, newFile])
```

Comment:
Can we ensure that we're only loading the same of any one sound here?  i.e. keep track in a hashmap or set of file cids, etc

Reply:
Yeah I left a comment about that in the commit, but haven't handled it yet